src/components/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,68 @@
+import React, {PropTypes} from 'react';
+import {themr} from 'react-css-themr';
+import classnames from 'classnames';
+import {randomId} from 'dx-util/src/string/string';
+
+import Icon from '../Icon/Icon.jsx';
+import defaultCheckboxIcon from './img/icon-checkbox-tick.svg';
+
+export const CHECKBOX = Symbol('Checkbox');
+
+const Checkbox = ({theme, ...props}) => {
+	const id = props.id || randomId('control-checkbox');
+	const labelClassName = classnames(theme.container__label, {
+		[theme.container__disabled]: props.disabled
+	});
+	return (
+		<span className={theme.container}>
+			<input {...cleanProps(props)}
+					id={id} type="checkbox"
+					checked={props.checked}
+					onChange={props.onChange}
+					className={theme.container__input} />
+			<label htmlFor={id} className={labelClassName}>
+				<span className={theme.container__view}>
+					<span className={theme.container__checkboxIcon}>
+						<Icon name={props.checkboxIconName} />
+					</span>
+				</span>
+				{props.children}
```

Comment:
Why do use wrap in a `label` accept `children` here? 
Checkbox is supposed to be used like this:
```javascript
<label for="my_checkbox">
custom text
<Checkbox {...somePropsHere}/>
</label>
```